bpf/bpf_lxc.c

```diff
@@ -62,6 +62,7 @@ struct bpf_elf_map __section_maps CT_MAP_TCP6 = {
 	.size_value	= sizeof(struct ct_entry),
 	.pinning	= PIN_GLOBAL_NS,
 	.max_elem	= CT_MAP_SIZE_TCP,
+	.flags		= DEFAULT_MAP_FLAGS,
```

Comment:
LRU maps in non-prealloc mode cannot be created in the kernel.

Reply:
Right, I was testing on 4.9 without LRU to get started. I've now made it conditional.